lib/swim/ping-req-recvr.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
This is an explicit ping-req handler. I named it recvr because of its relationship to "sender".